src/main/kotlin/org/elm/workspace/ElmCLI.kt

```diff
@@ -11,22 +10,22 @@ class ElmCLI(private val elmExecutablePath: Path) {
 
     // TODO [kl] allow the caller to specify the main entry point Elm file
 
-    fun make(owner: Disposable, elmProject: ElmProject): ProcessOutput {
```

Comment:
This parameter will need to be added back in once the `execute` method that takes a Disposable is restored.